service/collector/servicequota.go

```diff
@@ -60,8 +64,9 @@ func NewServiceQuota(config ServiceQuotaConfig) (*ServiceQuota, error) {
 	}
 
 	v := &ServiceQuota{
-		helper: config.Helper,
-		logger: config.Logger,
+		awsAPIcache: cache.NewFloat64Cache(time.Minute * 60),
```

Comment:
I would also like to have some explanations of why we thing 60 minutes are a good value for this. Just to be able to still reason about it in half a year.